### DISCUSSION TITLE
Altitude does not change apparent altitude: why?

### DISCUSSION THREAD

**Discussion by Runmartinrun** [Stellarium/stellarium#1704] [Q&A]
Dear all,
If I change altitude of the observer in Stellarium 0.13.2, the apparent altitude of bodies does not change, while it should change, because atmospheric refraction depends on the observer's altitude. Why?

Also, do you know what atmospheric model Stellarium uses to compute the altitude correction (the difference between altitude and apparent altitude) and to which level of accuracy (arcminute, fraction of arcminute, ...?) one should consider this correction to be correct, given that one provides the correct parameters (temperature, pressure, ...)?

Thanks!

**Comment by gzotti** [comment]
If you are still using 0.13.2, you are missing accuracy and many other features. Are you also still running Windows 8.0? Or Vista?

The atmospheric density does not directly change from altitude in Stellarium. You can play with pressure in the atmosphere settings. The altitude influences parallax e.g. for Solar eclipses or stellar occultations. 

Stellarium's User Guide tells you what some developers created, could still remember or have reverse engineered to find out these questions.

**Reply by Runmartinrun** [reply]
Thank you, I will try a newer version then, but I am running an old OS X version, so I will see what will work on my computer. 
If atmospheric density does not directly change with altitude, then the refraction correction from Stellarium does not seem to be correct.

**Reply by gzotti** [reply]
Make it better then. We also don't compute temperature inversions, high/low pressure weather, etc.
